apple-pie/styles/style.css

```diff
@@ -21,4 +21,56 @@ body {
 }
 
 /* STYLES */
-/* Write your CSS below */
\ No newline at end of file
+/* Write your CSS below */
+body{
+    margin-left: 60px;
```

Comment:
This margin property in the body is why your header image doesn't have the screen width